ui/apps/dotcms-ui/src/app/view/components/dot-portlet-base/dot-portlet-base.stories.ts

```diff
@@ -98,13 +98,13 @@ export const BasicActions = () => ({
             primary: [
                 {
                     label: 'Save',
-                    command: (e) => {
-                        console.log(e);
+                    command: () => {
+                        //console.log(e);
```

Comment:
I think we can remove these `console.logs`/`commets`.

Reply:
removed